devito/passes/iet/parpragma.py

```diff
@@ -426,6 +428,86 @@ def make_parallel(self, iet):
         return self._make_parallel(iet)
 
 
+class PragmaTransfer(Pragma, Transfer):
+
+    """
+    A data transfer between host and device expressed by means of one or more pragmas.
+    """
+
+    def __init__(self, callback, function, imask=None, arguments=None):
+        super().__init__(callback, arguments)
+
+        self._function = function
+        self._imask = imask
+
+    @property
+    def function(self):
+        return self._function
+
+    @property
+    def imask(self):
+        return self._imask
+
+    @cached_property
+    def sections(self):
```

Comment:
not really new code here -- nearly a copy-paste from code that previously was in `langbase.py`